docs/user-management/index.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 1
+title: User Management
+---
+
+This section explains how to view users and interpret the provided information. Refer to [Authentication](../authentication.md) for information about accessing your Macrometa account.
+
+## View Users
+
+Once you log in to a Macrometa account, you can view all users if you have read permissions for the GeoFabric.
+
+1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
+1. Click **Account**.
+
+Macrometa displays the **Users** tab which lists all user accounts. Type a query in the **Filter users** field to narrow the search and find a user by their ID, email address, tenant, or username.
+
+You can click a user's ID to view more details about the user. The following columns show information about each user.
+
+### ID
+
+The unique ID for the user. This value is automatically generated.
+
+### Email Address
+
+The email address associated with the user.
+
+### Tenant
+
+The tenant on which the user is hosted.
+
+### Username
+
+The name chosen when the user was created.
+
+### Status
+
+Users are created as active or inactive, and you can activate or deactivate them at a later time.
+
+## View API Keys
+
+1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
+1. Click **Account** and then the **API Keys** tab.
+
+You can click the ID of an API key to view its details. The following columns show information about each API key.
```

Comment:
I would put this as a third step.

Not sure that View API Keys belongs in User Management, though. Seems like this should be with Authentication.

Reply:
Great suggestions!  Condensing the info into optional steps makes this article a lot more concise and effective. Thanks!